stable/prometheus-operator/templates/prometheus/ingressThanosSidecar.yaml

```diff
@@ -0,0 +1,48 @@
+{{- if and .Values.prometheus.enabled .Values.prometheus.thanosIngress.enabled }}
+{{- $serviceName := printf "%s-%s" (include "prometheus-operator.fullname" .) "prometheus" }}
+{{- $thanosPort := .Values.prometheus.thanosIngress.servicePort -}}
+{{- $routePrefix := list .Values.prometheus.prometheusSpec.routePrefix }}
+{{- $paths := .Values.prometheus.thanosIngress.paths | default $routePrefix -}}
+apiVersion: extensions/v1beta1
+kind: Ingress
+metadata:
+{{- if .Values.prometheus.thanosIngress.annotations }}
+  annotations:
+{{ toYaml .Values.prometheus.thanosIngress.annotations | indent 4 }}
+{{- end }}
+  name: thanos-gateway-{{ .Release.Name }}
```

Comment:
This doesn't seem to be following naming conventions used elsewhere.

Reply:
Would this be more preferred? 
```
name: {{ template "prometheus-operator.fullname" . }}-thanos-gateway
```